### PR TITLE
BD-56 feat: edit profile

### PR DESCRIPTION
**Pull request by KateTernovskaya**

### CODE REVIEW

File: components/profile/GeneralInformationForm.tsx [hunk 1]
```diff
@@ -65,6 +65,7 @@ type FormFields = z.infer<ReturnType<typeof ProfileSchema>>
 export const GeneralInformationForm = () => {
   const { t } = useTranslation()
   const { data } = authApi.useAuthMeQuery()
```
Comment (sfgardev) [inline components/profile/GeneralInformationForm.tsx:67]: we can get rid of authMe and take userName from profileData

File: components/profile/GeneralInformationForm.tsx [hunk 2]
```diff
@@ -86,13 +87,13 @@ export const GeneralInformationForm = () => {
...
-      lastName: '',
-      region: '',
+      aboutMe: profileData?.aboutMe || '',
+      city: profileData?.city || '',
+      country: profileData?.country || '',
+      dateOfBirth: profileData?.dateOfBirth || '',
+      firstName: profileData?.firstName || '',
+      lastName: profileData?.lastName || '',
+      region: profileData?.region || '',
       userName: data?.userName || '',
```
Comment (sfgardev) [inline components/profile/GeneralInformationForm.tsx:97]: use profleData.userName

**Review by sfgardev — CHANGES_REQUESTED** [review]

File: components/profile/GeneralInformationForm.tsx (hunk 1 above)
Comment (KateTernovskaya) [reply components/profile/GeneralInformationForm.tsx]: done

File: components/profile/GeneralInformationForm.tsx (hunk 2 above)
Comment (KateTernovskaya) [reply components/profile/GeneralInformationForm.tsx]: done

**Pull request merged**